projects/router/routes.js

```diff
@@ -9,3 +9,14 @@ FlowRouter.route("/projects/add", {
     BlazeLayout.render("mainLayout", {main: "addProject"});
   }
 });
+
+JsonRoutes.add("get", "/projects/:id", function (req, res, next) {
+  var id = req.params.id;
+
+  JsonRoutes.sendResult(res, 200, Projects.findOne(id));
+});
+
+JsonRoutes.add("get", "/projects", function (req, res, next) {
+
```

Comment:
Add a comment here describing the general intention of this code in human language. While the JavaScript may be relatively easy to read, providing a human language description of the intention has the following benefits:

* Clearly explains the purpose of code to other developers
* doesn't require a human to parse JavaScript, unless they are debugging/enhancing the code
* provides an opportunity to reflect on the purpose of the code, possibly illuminating logical flaws
  * e.g. variable names that don't reflect their reference
  * logical errors in the line of code